db/db_impl/db_impl_secondary.h

```diff
@@ -172,6 +172,12 @@ class DBImplSecondary : public DBImpl {
     return Status::NotSupported("Not supported operation in secondary mode.");
   }
 
+  // For the secondary instance, this function only get live files
+  // and doesn't flush memtable.
+  virtual Status GetLiveFilesStorageInfo(
```

Comment:
Remove `virtual` and keep only `override`, refer to https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-override.html.